package.json

```diff
@@ -74,7 +74,7 @@
     "gen-esm-wrapper": "^1.1.0",
     "husky": "^8.0.3",
     "mocha": "^10.2.0",
-    "mongodb": "^5.4.0",
+    "mongodb": "^6.7.0",
```

Comment:
This is already close to the maximal breaking change we could do for this work, we might as well remove `abort-controller` too.

Reply:
you mean our `OIDCAbortSignal`, right?

Reply:
No – I mean `abort-controller` 🙂

Reply:
ah, i noticed it in deps! will remove it

Reply:
removed in [67ede2b](https://github.com/mongodb-js/oidc-plugin/pull/191/commits/67ede2b35a9def9bcfb528ec10e4333ffa30004f)